Sources/BraintreeCoreSwift/BTAnalyticsEvent.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+/// Encapsulates a single analytics event
+struct BTAnalyticsEvent {
+    var kind: String
+    var timestamp: Double
```

Comment:
We could likely update this in next major version/this PR if we wanted them to be consistent. I am not sure if there are any implications to making this update now that I may not be thinking of. Curious if @scannillo knows of any reasons we may want to wait to change this that I am not thinking of?

Reply:
I don't think there's a rush to change - I just wanted to make note.

Reply:
It's up to us! I don't know of a compelling reason _not_ to change it now if changing it will allow us to be better aligned across the SDKs unless I am missing some context for why it needed to be a Double initially.

Reply:
It looks like [we were using `Uint64`](https://github.com/braintree/braintree_ios/blob/8964e496f38487439a70dc1d7fd0c1e9a91508f8/Sources/BraintreeCore/BTAnalyticsService.m#L246) in Objective-C and we're using a `long` in Java. We should hold off on changing this to a string, until we're ready for FPTI migration. We want to keep consistent with other platform analytics using Arachne.

Also, I think this might have been used for easy conversion to Redshift's DATE or TIMESTAMP type.

Reply:
Epoch timestamps will only be in whole numbers, so we don't need to use a floating-point type (ex: Double or Float). `UInt64` I think will be our best bet. [This site](https://www.tutorialspoint.com/swift/swift_data_types.htm) does a good job of explaining Swift's numeric types.

Reply:
It looks like doing `Date().timeIntervalSince1970 * 1000` without any cast, defaults to `Double` and spits out something like `1667488950096.996` 🤔. I thought that Epoch times in seconds are 10-digit INTs, milliseconds are 13-digit INTs and microseconds are 16-digit INTs.

I double checked, and on master, we have something like `1667489463800` (10 digit INT), without decimals.

I think doing something like `UInt64(Date().timeIntervalSince1970 * 1000)`, would match with our Obj-C implementation.